Mimus/Source/Verification/VerificationHandler.swift

```diff
@@ -42,14 +48,29 @@ internal class VerificationHandler {
                 line: testLocation.line)
     }
 
+    private func assertAtMost(callIdentifier: String, times: Int, matchCount: Int, differentArgumentsMatchCount: Int, testLocation: TestLocation) {
+        guard matchCount > times else {
+            return
+        }
+
+        var message = "No call with identifier \(callIdentifier) was captured"
+        if matchCount > 0 {
+            message = "Call with identifier \(callIdentifier) was recorded \(matchCount) times, but expected at most \(times)"
+        }
+
+        XCTFail(message, file: testLocation.file, line: testLocation.line)
+    }
+
     private func assertAtLeast(callIdentifier: String, times: Int, matchCount: Int, differentArgumentsMatchCount: Int, testLocation: TestLocation) {
         guard matchCount < times else {
             return
         }
 
         var message = "No call with identifier \(callIdentifier) was captured"
-        if differentArgumentsMatchCount > 0 {
-            message = "Call with identifier \(callIdentifier) was recorded, but arguments didn't match"
+        if matchCount > 0 {
+            message = "Call with identifier \(callIdentifier) was recorded \(matchCount) times, but expected at least \(times)"
+        } else if differentArgumentsMatchCount > 0 {
+            message = "Call with identifier \(callIdentifier) was recorded \(differentArgumentsMatchCount) times, but arguments didn't match"
```

Comment:
This actually made me thinking that the reporting here might not be ideal. 

Specifically for following scenario I see a usability issue: 

1. Expect two calls
2. Two calls are recorded, but one has different arguments.

In this case the message will be `Call with identifier xyz was recorded 1 times, but expected at least 2`. Would be nice to handle this too, albeit I think it's fine to merge 👍 I'll add a separate issue for this.